.ci/matrix_job.yaml

```diff
@@ -342,12 +357,27 @@ steps:
   - name: Test
     enable: ${do_test}
     containerSelector:
-      - "{name: 'test'}"
+      - "{name: 'sockperf'}"
+    agentSelector:
+      - "{nodeLabel: 'skip-agent'}"
+    run: |
+      [ "x${do_test}" == "xtrue" ] && action=yes || action=no
+      env WORKSPACE=$PWD TARGET=${flags} jenkins_test_run=${action} WORKER_THREADS=false ./contrib/test_jenkins.sh
+    parallel: false
+    onfail: |
+      ./.ci/artifacts.sh
+    archiveArtifacts-onfail: |
+      jenkins/**/arch-*.tar.gz
+
+  - name: Test (Worker thread mode)
+    enable: ${do_test}
+    containerSelector:
+      - "{name: 'sockperf-worker-threads'}"
     agentSelector:
       - "{nodeLabel: 'skip-agent'}"
     run: |
       [ "x${do_test}" == "xtrue" ] && action=yes || action=no
-      env WORKSPACE=$PWD TARGET=${flags} jenkins_test_run=${action} ./contrib/test_jenkins.sh
+      env WORKSPACE=$PWD TARGET=${flags} jenkins_test_run=${action} WORKER_THREADS=true ./contrib/test_jenkins.sh
```

Comment:
@BasharRadya - keep me honest - it's numeric, not boolean, right?
we also agreed to have 2 worker threads for more code coverage?

Reply:
ahh silly me - it's @NirWolfer mechanism for transferring info.
Nir, consider either adding jenkins_test_workers_run like everyone did before you.
or alternatively, introduce a new mechanism but let it start with jenkins_